docs/dev-setup.md

```diff
@@ -76,6 +76,19 @@ The frontend is run with `yarn`.  We recommend running the backend via `eclipse`
 1. Run `./gradlew build` to download all dependencies and build the project.
    - Some tests will fail if you are using SQLite, because it has a bad implementation of some timezone stuff. You'll need to use MSSQL to see all the tests passing.
 
+#### Initial Setup Test Database
+After successfully creating and building the MSSQL Docker container it is posisble to create a dedicated container for testing. Use the `-PtestEnv` property to access the test environment settings in `gradle`.
+1. Create the MSSQL Docker container and test database `./gradlew -PtestEnv dockerCreateDB`
+1. Start the MSSQL Docker container: `./gradlew -PtestEnv dockerStartDB`
+1. Wait until the container is fully started, then run `./gradlew -PtestEnv dbMigrate`
+1. Run `./gradlew -PtestEnv build` to download all dependencies and build the project.
+1. Seed initial data - MSSQL: `./gradlew -PtestEnv dbLoad`.
+
+##### Override Default Gradle Settings
+Override the default gradle settings if you want to run your tests on a different database:
+   1. Open a command line in the `anet` directory that was retrieved from github.
+   1. Create a new empty file at `localTestSettings.gradle`. (`touch localTestSettings.gradle` on linux/mac).  This will be a file for all of your local test settings and passwords that should not be checked into GitHub.
```

Comment:
"GitHub" capitalisation is inconsistent; other (non-URL) occurrences use "github" in all lower-case.

Reply:
Inconsistency is copied from the original text for: Set up Gradle: 'localSettings.gradle'.
Thereby, GitHub is also used in heading GitHub Pages and GitHub repository.

Should I update the inconsistency everywhere? Or only the dev-setup, but then also the original localSettings text?